hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/INode.java

```diff
@@ -993,15 +993,13 @@ public long getNsDelta() {
     private final QuotaDelta quotaDelta;
 
     /**
-     * @param bsps
- *          block storage policy suite to calculate intended storage type
```

Comment:
How about just adding the same blanks on line 997 and below wrong format lines as line 996, and do not change other lines.

Reply:
Thanks @ferhui for the comment and review.
I will update it later.
The new style will look like this:
![image](https://user-images.githubusercontent.com/6416939/140021341-f0016022-613a-4de2-a658-a63c336b2b14.png)